gpii/node_modules/deviceReporter/test/configs/live.json

```diff
@@ -0,0 +1,70 @@
+{
+    "type": "gpii.tests.deviceReporter.config.live",
+    "options": {
+        "components": {
+          "deviceReporterMock": {
+              "type": "gpii.test.integration.deviceReporterAware",
+              "options": {
+                  "mockDeviceReporters": {
+                      "gpii.packageKit.find": {
+                          "mockFunc": "findNameInList",
+                          "defaults": {
+                              "gradeNames": "fluid.function",
+                              "argumentMap": {
+                                  "name": 0
+                              }
+                          }
+                      }
+                  },
+                  "deviceReporters": {
+                      "gpii.packageKit.find": {
+                          "expectInstalled": [
+                              "rainbowIslands",
+                              "barbarian",
+                              "greenBeret"
```

Comment:
This is up to you, but perhaps change this to a JSON 5 file, and explicitly comment out the greenBeret solution describing that this is reported via alwaysInstalled - so it's clear that you're testing that functionality